src/transactions/sendBtcTransaction.ts

```diff
@@ -11,7 +21,7 @@ export const sendBtcTransaction = async (options: SendBtcTransactionOptions) =>
     throw new Error('No Bitcoin wallet installed');
   }
 
-  const { recipients, senderAddress } = options.payload;
+  const { recipients, senderAddress, network, message } = options.payload;
   if (!recipients || recipients.length === 0) {
```

Comment:
we need to also add a check on the recipients's type before attempting to serialize it and pass it to the extension